test/ui_test_suite.rb

```diff
@@ -4947,8 +4958,8 @@ def teardown
           file_type.send_keys('Metadata')
         when bam_file
           file_type.send_keys('BAM')
-          species_dropdown = form.find_element(:id, 'study_file_taxon_id')
-          species_dropdown.send_keys('mouse') # from lib/assets/default_species_assemblies.txt
+          # TODO: Determine where else this is needed, apply there.
+          select_option_from_dropdown(form, 'study_file_taxon_id', 'mouse')
```

Comment:
Really like this.  We should create a ticket in the backlog to apply this to all select boxes in the UI test suite.

Reply:
Done! https://broadinstitute.atlassian.net/browse/SCP-1572